src/commands/moderation.rs

```diff
@@ -155,55 +155,61 @@ pub async fn unlock(ctx: Context<'_>) -> Result<(), Error> {
     .await
 }
 
-/// Unmute a member.
+/// Unmute a user.
 #[poise::command(slash_command)]
 pub async fn unmute(
     ctx: Context<'_>,
-    #[description = "The member to unmute"] member: Member,
+    #[description = "The user to unmute"] user: UserId,
```

Comment:
Technically we are unmuting a member because you can only mute or unmute members of a server even though `UserId` is used here. For that reason, comments and descriptions for slash commands should still refer to members even though internally they handle them as users.

Reply:
Done

Reply:
> Technically we are unmuting a member because you can only mute or unmute members of a server even though `UserId` is used here. For that reason, comments and descriptions for slash commands should still refer to members even though internally they handle them as users.

Keep in mind that with this PR, we will be able to update the database even if the user is not in the guild.
This is why I initially changed the parameter name to `user`.